<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="tracker">
    <link name="world"/>
    <!-- <gazebo>
        <gravity>false</gravity>
    </gazebo> -->
    <gazebo reference="tracking_dot">
        <!-- must come before the link definition (major ???) -->
        <material>Gazebo/Red</material>
    </gazebo>
    <link name="tracking_dot">
        <visual>
            <origin xyz="0 0 0.015"/>
            <geometry>
                <box size="0.03 0.03 0.03"/>
            </geometry>
            <material name="red">
                <color rgba="1.0 0 0 1.0"/>
            </material>
        </visual>
        <!-- <collision>
        <origin xyz="0 0 0.015" />
        <geometry>
          <box size="0.03 0.03 0.03" />
        </geometry>
      </collision> -->
        <inertial>
            <mass value="0.001"/>
            <inertia ixx="1"
                     ixy="0.00"
                     ixz="0.00"
                     iyy="1"
                     iyz="0.00"
                     izz="1"/> <!-- ${base_mass_kg/12*(base_x_m**2 + base_z_m**2)} -->
        </inertial>
    </link>

    <joint name="tracker" type="prismatic">
        <axis xyz="0 1 0"/>
        <parent link="world"/>
        <child link="tracking_dot"/>
        <origin rpy="0 0 0" xyz="0 0 0"/>
        <limit lower="-100" upper="100" effort="1000" velocity="1000"/>
    </joint>

</robot>